src/components/InputButton/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Input, Button } from '@ht6/react-ui';
+import { ReactNode } from 'react';
+import cx from 'classnames';
+import { container, field, button } from './InputButton.module.scss';
+
+export interface InputButtonProps {
+  className?: string;
+  label: string;
+  name: string;
+  status?: {
+    state: "success" | "error";
+    text?: ReactNode;
+  };
+  onSubmit: (event: any) => any;
+  onChange: (event: any) => any;
+}
+
+function InputButton({
+  className,
+  label,
+  name,
+  status,
+  onSubmit,
+  onChange
+}: InputButtonProps) {
+  return (
+    <form onSubmit={onSubmit}>
+      <div className={container}>
+        <Input outlineColor="primary-1" className={cx(field, className)} hideLabel status={status} label={label} name={name} onChange={onChange}/>
+        <Button className={cx(button, className)} type="submit">{name}</Button>
```

Comment:
This is a bit weird, it would make more sense to apply the className on the parent. If you want classNames specifically for those elements, use a another prop (ex. `inputClassName` or `buttonClassName`)

Reply:
Also lets add some more control to those components so we can pass dynamic props to them respectively via a custom prop

Reply:
What do you mean by the second comment?

Reply:
What do you mean by the second comment?

Reply:
like you can have a prop like `inputProps` and then pass it into the input as `{...inputProps}`